src/analytics/Properties.tsx

```diff
@@ -1296,6 +1296,7 @@ interface NftsEventsProperties {
     origin: NftOrigin
     error?: string
   }
+  [NftEvents.nft_gallery_screen_open]: undefined
```

Comment:
what do you think about adding the number of nft's a user has to this event property? it might be interesting to understand if our users are holding many or few nft's?

Reply:
bump on this? :)